kasa/cli.py

```diff
@@ -90,19 +90,62 @@ def wrapper(message=None, *args, **kwargs):
 pass_dev = click.make_pass_decorator(Device)
 
 
-class ExceptionHandlerGroup(click.Group):
-    """Group to capture all exceptions and print them nicely.
+def CatchAllExceptions(cls):
+    """Capture all exceptions and prints them nicely.
 
-    Idea from https://stackoverflow.com/a/44347763
+    Idea from https://stackoverflow.com/a/44347763 and
+    https://stackoverflow.com/questions/52213375
     """
 
-    def __call__(self, *args, **kwargs):
-        """Run the coroutine in the event loop and print any exceptions."""
-        try:
-            asyncio.get_event_loop().run_until_complete(self.main(*args, **kwargs))
-        except Exception as ex:
-            echo(f"Got error: {ex!r}")
+    def _handle_exception(cmd, info_name, exc):
+        if isinstance(exc, click.ClickException):
+            raise
+        echo(f"Kasa:: Command line: {info_name} {cmd._masked_args}")
+        echo(f"Kasa:: Raised error: {exc}")
```

Comment:
I'd avoid styling here, and just use "Got error" or "Raised error" if that's a better wording.

Reply:
Done